<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-A5D09V8S" failures="1" tests="3" name="Test" time="30.81" errors="0" timestamp="2025-07-20T23:47:58 IST">
  <testcase classname="tests.FourTest" name="testFour" time="12.444">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at tests.FourTest.testFour(FourTest.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- testFour -->
  <testcase classname="tests.TwoTest" name="testTwo" time="8.607"/>
  <testcase classname="tests.ThreeTest" name="testThree" time="9.008"/>
</testsuite> <!-- Test -->
